Trumpf.Coparoo.Web/Controls/Table/TableCell.cs

```diff
@@ -0,0 +1,31 @@
+﻿namespace Trumpf.Coparoo.Web.Controls.Table
+{
+    using OpenQA.Selenium;
+    using Trumpf.Coparoo.Web;
+
+    /// <summary>
+    /// Table data cell object.
+    /// </summary>
+    public class TableCell : ControlObject
+    {
+        /// <summary>
+        /// Gets the search pattern.
+        /// </summary>
+        protected override By SearchPattern => By.XPath(".//th|.//td");
+
+        /// <summary>
+        /// Searches for a control object like provided and returns it.
+        /// </summary>
+        public T As<T>() where T : ControlObject => Find<T>();
+
+        /// <summary>
+        /// Returns true if node element is header cell, otherwise false.
+        /// </summary>
+        public bool IsHeaderCell => Node.TagName.Equals("th");
+
+        /// <summary>
+        /// Returns true if node element is data cell, otherwise false.
+        /// </summary>
+        public bool IsDataCell => Node.TagName.Equals("td");
```

Comment:
If this is a cell in the "body"-part of the table I suggest to rename the property to `IsBodyCell`. The term "Data" appears only here right now.

Reply:
I think we should leave it as `IsDataCell` since `<td>` elements can also occur in the table foot section. I also think it's more intuitive since a `<th>` element usually contains **h**-eader information and a `<td>` element contains **d**-ata.